slices/ca-certificates-java.yaml

```diff
@@ -28,6 +28,12 @@ slices:
       /etc/ssl/certs/java/:
       /usr/share/ca-certificates-java/ca-certificates-java.jar:
 
+  data:
+    essential:
+      - ca-certificates_data-with-certs
```

Comment:
do you really need the certs? is it because you're going to run the `ca-certificates-java.jar` postinst? If so, then I would suggest calling this slice `data-with-certs` too, to be consistent with `ca-certificates`.

Reply:
Yes, those are needed for ca-certificates-java.jar. 
Renamed the slice.